test/resources/components/component-with-inject-composition.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>{{ fromMount }}</div>
+</template>
+
+<script>
+import { createComponent, inject } from '@vue/composition-api'
+
+export default createComponent({
+  name: 'component-with-inject-composition',
+  setup: () => {
+    const fromMount = inject('fromMount')
+    const setInBeforeCreate = 'created'
```

Comment:
What is the purpose of this `setInBeforeCreate` property? We dont use it in the assertions.

Reply:
It was previously used in a similar Resource test component ([here](https://github.com/vuejs/vue-test-utils/blob/be4033e2e22440255441a9ab9e1fe717a2c456a0/test/resources/components/component-with-inject.vue)) so I think he was trying to maintain parity with the previous example. (setup now maps to "beforeCreate" and "created" seen [here](https://github.com/vuejs/composition-api-rfc/blob/master/api.md#lifecycle-hooks))

We should test for it in the assertions.